packages/api-explorer/src/index.jsx

```diff
@@ -209,6 +209,11 @@ class ApiExplorer extends React.Component {
     const NewGlossaryTermsContext = utils.GlossaryContext;
     /* eslint-enable global-require */
 
+    const onAuthGroupChange = auth => {
```

Comment:
Instead of doing this here, can you move the following line into the top of the `onAuthGroupChange` function so this supplementary behavior is contained in there? And then in the `onAuthGroupChange` call on line 251 that can remain as `this.onAuthGroupChange`.

Reply:
Also shouldn't need to verify that `this.props` exists since that's always present.